artwork/README.rst

```diff
@@ -1,5 +1,4 @@
-:orphan:
```

Comment:
I think it was intentional to have this marker.

Reply:
It [looks](https://github.com/scrapy/scrapy/pull/2507#issue-102643392) like it was added to avoid Sphinx picking it up, but it also [looks](https://travis-ci.org/scrapy/scrapy/jobs/637250957) like it is not being picked up after the change, so I’m OK with the change.